rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2464,6 +2464,52 @@ void voteFederationChange_addWrongFederatorPublicKey_returnsGenericResponseCode(
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorMultiKey_returnsExpectedPendingFederationSizeResponseCode() {
```

Comment:
`_returnsSuccessful`. And pending federation size response code does not exist

Reply:
Updated.